### PR TITLE
Issue10

### PR DESCRIPTION
**Pull request by EN-Chou**
This commit works on issue #9 and #10:
#10: equations are labelled automatically instead of given value. 

Also, this commit added `#` in between equations and texts.

### CODE REVIEW

File: projection.tex [hunk 1]
```diff
@@ -80,133 +80,167 @@ \section{Govern equations for 2D incompressible flow}
 
 For simplicity, suppose $\rho=1$.
 Momentum equation in x-direction:
-\begin{align*}
-\frac{\partial u}{\partial t}+u \frac{\partial u}{\partial x}+v \frac{\partial u}{\partial y}=-\frac{\partial p}{\partial x}+ \nu (\frac{\partial^2 u}{\partial x^2}+\frac{\partial^2 u}{\partial y^2})
-\tag{1}
-\end{align*}
+\begin{align}
```
Comment (yungyuc) [inline projection.tex:83]: You should use a `%` here too.

File: projection.tex [hunk 2]
```diff
@@ -80,133 +80,167 @@ \section{Govern equations for 2D incompressible flow}
...
-\frac{\partial u}{\partial x}+\frac{\partial v}{\partial y}=0
-\tag{3}
-\end{align*}
-
+%
+\begin{align}
+\frac{\partial u}{\partial x}+\frac{\partial v}{\partial y}=0 \label{e:continuity}
+\end{align}
+%
 \subsection{ Vector form}
```
Comment (yungyuc) [inline projection.tex:103]: Please fix the whitespace before `V`.

**Review by yungyuc — CHANGES_REQUESTED** [review]
LGTM except the minor issues.  Please leave a comment once you fix them.

**Comment by EN-Chou** [comment]
@yungyuc I have fixed them.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
> @yungyuc I have fixed them.

It's great.  Thanks.

**Comment by yungyuc** [comment]
Please close the two associated issues.

**Pull request merged**